src/adminjs.ts

```diff
@@ -306,7 +306,7 @@ class AdminJS {
     if (src[0] === '/') {
       filePath = src
     } else {
-      filePath = relativeFilePathResolver(src, /Function\.bundle/)
+      filePath = relativeFilePathResolver(src, /.*\.{1}bundle/)
```

Comment:
This line failed for me because in my case the name wasn't `Function.bundle` but `Proxy.bundle`. The new regex should allow:
- `Function.bundle`
- `Proxy.bundle`
- `Anything.bundle`
- `.bundle`

CC @hank121314

Reply:
Thanks for fixing this issue!
But I am wondering why `Admin.bundle` will convert into `Proxy.bundle` .... 😓